common/api/imodeljs-backend.api.md

```diff
@@ -425,8 +425,8 @@ export class BriefcaseDb extends IModelDb {
     readonly onBeforeClose: BeEvent<() => void>;
     readonly onChangesetApplied: BeEvent<() => void>;
     static readonly onCreated: BeEvent<(_imodelDb: BriefcaseDb) => void>;
-    static readonly onOpen: BeEvent<(_requestContext: AuthorizedClientRequestContext | ClientRequestContext, _briefcaseProps: BriefcaseProps) => void>;
-    static readonly onOpened: BeEvent<(_requestContext: AuthorizedClientRequestContext | ClientRequestContext, _imodelDb: BriefcaseDb) => void>;
+    static readonly onOpen: BeEvent<(_requestContext: ClientRequestContext | AuthorizedClientRequestContext, _briefcaseProps: BriefcaseProps) => void>;

```

Comment:
Have we filed an issue with api-extractor about this annoying bug?

Reply:
I thought the bug had gone away... I guess it still exists for certain Node versions and platforms.  @evanpreslar-bentley what node version are you on?  Can you reliable reproduce this?

Reply:
If I make the same changes to IModelHost.ts and IModelDb.ts on master branch, build, and extract-api, the markdown file is affected in the same way.

D:\dev\imodeljs>node -v
v12.16.2

D:\dev\imodeljs>npm -v
6.14.4